### PR TITLE
Edit USUM Wormhole guide

### PR DESCRIPTION
**Pull request by ShinySylveon04**
# Guide Details

## I want to (select one)

- [ ] Add a guide
- [x] Modify an existing guide

## Games affected

- USUM

## Type of RNG

- Wormhole RNG

## Language

- English

## Notes

This PR will be a draft not ready for review until PR #124 is merged and this guide updated afterwards

### CODE REVIEW

File: guides/Ultra Sun and Ultra Moon/Stationary Wormhole RNG.md [hunk 1]
```diff
@@ -12,59 +12,31 @@ subCategory: 'Custom Firmware'
 - [3DSRNGTool](https://github.com/wwwwwwzx/3DSRNGTool/releases)
 - PCalc
   - [Ultra Sun/Ultra Moon](https://pokemonrng.com/downloads/pcalc/pcalc-usum.zip)
-
-### Recommended reading/references
-
-- [Timeline Guide](https://www.pokemonrng.com/retail-usum-timeline)
-- [Timeline and Timeline 2.0 Guide](https://github.com/wwwwwwzx/3DSRNGTool/wiki/Gen7-Timeline-Calibration-%28PokeCalcNTR-Only%29)
-- [3DSRNGTool README](https://github.com/wwwwwwzx/3DSRNGTool/blob/master/README.md) - list of final screens before the Pokemon is generated
+- [3DSRNGTool README](https://github.com/wwwwwwzx/3DSRNGTool/blob/master/README.md#final-screen) - list of final screens before the Pokemon is generated
```
Comment (zaksabeast) [inline guides/Ultra Sun and Ultra Moon/Stationary Wormhole RNG.md:15]:
```suggestion
- [3DSRNGTool README](https://github.com/wwwwwwzx/3DSRNGTool/blob/master/README.md#final-screen) - list of final screens before Pokemon are generated
```

**Review by zaksabeast — CHANGES_REQUESTED** [review]
One tiny change, but otherwise good to go.

File: guides/Ultra Sun and Ultra Moon/Stationary Wormhole RNG.md [hunk 2]
```diff
@@ -12,59 +12,31 @@ subCategory: 'Custom Firmware'
...
   - [Ultra Sun/Ultra Moon](https://pokemonrng.com/downloads/pcalc/pcalc-usum.zip)
-
-### Recommended reading/references
-
-- [Timeline Guide](https://www.pokemonrng.com/retail-usum-timeline)
-- [Timeline and Timeline 2.0 Guide](https://github.com/wwwwwwzx/3DSRNGTool/wiki/Gen7-Timeline-Calibration-%28PokeCalcNTR-Only%29)
-- [3DSRNGTool README](https://github.com/wwwwwwzx/3DSRNGTool/blob/master/README.md) - list of final screens before the Pokemon is generated
+- [3DSRNGTool README](https://github.com/wwwwwwzx/3DSRNGTool/blob/master/README.md#final-screen) - list of final screens before Pokemon are generated
 
 ## Step 1: Finding your seed
```
Comment (RichardPaulAstley) [inline guides/Ultra Sun and Ultra Moon/Stationary Wormhole RNG.md:17]: I'd add a note that it works for UBs too ?

File: guides/Ultra Sun and Ultra Moon/Stationary Wormhole RNG.md [hunk 3]
```diff
@@ -85,9 +57,9 @@ My character mid fidget:
 
 So what happened?
 
-This is okay and normal as each 3ds/save has its own delay that you will need to find. The delay is the time it takes from when you initiate the encounter and when the Pokemon actually gets generated by the game. Let's learn how to do that!
+This is okay and normal as each 3DS/save has its own delay that you will need to find. The delay is the time it takes from when you initiate the encounter and when the Pokemon actually gets generated by the game. Let's learn how to do that!
 
-## Step 5: Finding your delay and hitting it
+## Step 4: Finding your delay and hitting it
 
 1. We can use the IVs, nature and PSV of the Pokemon we got to see what frame we actually hit. My target was 4973 and I hit 4969 (-4). So the delay I hit vs the default setting of the 3DSRNGTool is -4 or 152.
```
Comment (RichardPaulAstley) [inline guides/Ultra Sun and Ultra Moon/Stationary Wormhole RNG.md:65]: Well, it's not exactly where I want this note to be but whatever : 

In the step 2) when the subject of repeating it, it'd be important to add a note for the Legendaries : Being precise and doing the same type of input (direction / pressure) will help to have a stable delay. The idea is that the last input is a step done by the stick. So not really accurate. So it's important to always do the same kind of input.

**Review by RichardPaulAstley — CHANGES_REQUESTED** [review]

**Review by RichardPaulAstley — APPROVED** [review]

**Pull request merged**